pkg/scaffold/v1/resource/resource.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/gobuffalo/flect"
 )
 
+const GroupMatchRegex = "^[a-z -]+$"
```

Comment:
shouldn't this be `^[a-z-]+$` or `^[a-z\-]+$`?  Otherwise we'll include whitespace in the allowed set.

Reply:
Really tks. Good point. Change requested performed.